src/components/UserProfile/UserProfile.scss

```diff
@@ -162,6 +162,10 @@
   padding-right: 10px;
 }
 
+.profileEditButtonContainer {
+  margin-top: 20px;
+}
+
```

Comment:
Hi @KyreneAF,
I was trying to find a reference to `.profileEditButtonContainer` that has been created above. Could you help me locate the use of this css. 
Thanks for helping out!